src/Modal.svelte

```diff
@@ -1,7 +1,40 @@
-<script>
+<script lang="ts">
     export let showModal;
 
+    let messages: messageType[] = [];
```

Comment:
This should stay in the App.svelte, Modal.svelte should only contain modal logic.